templates/containers/what-are-containers.html

```diff
@@ -14,282 +14,250 @@
 
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-8">
-        <h2>How to secure container images</h2>
-        <p>
-          Canonical’s <a href="https://juju.is/cloud-native-kubernetes-usage-report-2021#selection-criteria-for-container-images">Kubernetes and cloud native operations report</a> showed that the best container images are secure, stable, and easy to use. While containerisation adds a layer of security by keeping workloads isolated from one another and the host system, it is not enough to secure an application.
-        </p>
-        <p>
-          Container images are made of software from many different projects and sources, not all equally well-maintained. Developers need to ask themselves what content can developers rely on today, tomorrow and for a long time thereafter. Particular attention should be paid to the base layer, specifically in selecting a frequently updated one with credible maintenance commitment, a large software ecosystem of well-maintained packages, and a great developer experience.
-        </p>
-        <p>
-          Canonical publishes a portfolio of base, runtime, and application images with an up to 10-year maintenance commitment. In collaboration with Docker, this content is publicly available on Docker Hub.
-        </p>
+  <section class="p-section u-fixed-width">
+    <hr class="p-rule" />
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
+        <h2>Docker vs. LXD: comparing container technologies</h2>
+      </div>
+      <div class="col">
         <p>
-          <a href="/security/docker-images">Secure your software supply chain with LTS Docker Images&nbsp;&rsaquo;</a>
+          Process containers (such as Docker) and machine containers (traditional <a href="/blog/what-are-linux-containers">linux containers</a>) are two types of container technologies that can address different use cases. Process containers typically contain a single application, while machine containers can contain one or more applications as well as their own operating system. Both container technologies share a kernel with the host operating system.
         </p>
       </div>
-      <div class="col-4 u-vertically-center u-hide--medium u-hide--small u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/25c42877-simplified-software-management-2.svg",
-                alt="",
-                width="150",
-                height="150",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
+    </div>
+    <div class="u-fixed-width">
+      <div class="p-media-container p-strip is-shallow u-darker-background col-3 u-align--center u-vertically-center">
+        <img width="834"
+             src="https://assets.ubuntu.com/v1/a09a1433-diagram%202.png"
+             alt=""
+             loading="lazy" />
+      </div>
+      <div class="p-strip is-shallow row--50-50 ">
+        <div class="col"></div>
+        <div class="col">
```

Comment:
```suggestion
      <div class="p-strip is-shallow row ">
        <div class="col-start-large-7 col-6">
```
You can you use an offset here instead of an empty col, see [here](https://vanillaframework.io/docs/patterns/grid#empty-columns) for more